changelog/update-move-applyThousandSeparator-method-to-client-utils

```diff
@@ -0,0 +1,5 @@
+Significance: patch
```

Comment:
Let's keep a single changelog file for a PR and update the changes there.

Reply:
Thanks. I have removed the previous changelog entry.